packages/db/prisma/schema/user.prisma

```diff
@@ -0,0 +1,40 @@
+model User {
```

Comment:
# I THINK IT IS WORKING @bautistaaa 😂

Reply:
nvm 💀💀
<img width="1502" alt="image" src="https://github.com/user-attachments/assets/90b1da14-df29-4a85-9566-268f3b7ed14d">